modules/integration/tests-platform/tests-clustering/src/test/java/org/wso2/mb/platform/tests/clustering/slot/QueueClusterRdbmsSlotTestCase.java

```diff
@@ -0,0 +1,149 @@
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+package org.wso2.mb.platform.tests.clustering.slot;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import org.wso2.andes.configuration.AndesConfigurationManager;
+import org.wso2.carbon.andes.stub.AndesAdminServiceBrokerManagerAdminException;
+import org.wso2.carbon.andes.stub.admin.types.Queue;
+import org.wso2.carbon.authenticator.stub.LoginAuthenticationExceptionException;
+import org.wso2.carbon.automation.engine.context.AutomationContext;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.mb.integration.common.clients.AndesClient;
+import org.wso2.mb.integration.common.clients.configurations.AndesJMSConsumerClientConfiguration;
+import org.wso2.mb.integration.common.clients.configurations.AndesJMSPublisherClientConfiguration;
+import org.wso2.mb.integration.common.clients.operations.clients.AndesAdminClient;
+import org.wso2.mb.integration.common.clients.operations.utils.AndesClientConstants;
+import org.wso2.mb.integration.common.clients.operations.utils.AndesClientUtils;
+import org.wso2.mb.integration.common.clients.operations.utils.ExchangeType;
+import org.wso2.mb.platform.common.utils.MBPlatformBaseTest;
+import org.xml.sax.SAXException;
+
+import javax.xml.stream.XMLStreamException;
+import javax.xml.xpath.XPathExpressionException;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.rmi.RemoteException;
+
+import static org.testng.Assert.assertTrue;
+
+/**
+ * Test class to test successful message delivery using RDBMS slot implementation
+ */
+
+public class QueueClusterRdbmsSlotTestCase extends MBPlatformBaseTest {
+
+	/**
+	 * Prepare environment for test
+	 * @throws LoginAuthenticationExceptionException
+	 * @throws IOException
+	 * @throws XPathExpressionException
+	 * @throws URISyntaxException
+	 * @throws SAXException
+	 * @throws XMLStreamException
+	 */
+	@BeforeClass(alwaysRun = true)
+	public void init()
+			throws LoginAuthenticationExceptionException, IOException, XPathExpressionException,
+			       URISyntaxException, SAXException, XMLStreamException,
+			       AndesAdminServiceBrokerManagerAdminException {
+		super.initCluster(TestUserMode.SUPER_TENANT_ADMIN);
+		super.initAndesAdminClients();
+
+		String randomInstanceKey = getRandomMBInstance();
+
+		AndesAdminClient tempAndesAdminClient = getAndesAdminClientWithKey(randomInstanceKey);
+
+	}
```

Comment:
If this test case is for RDBMS slot replacement instead of hazelcast maps? Shouldnt we add the configuration modification(Like enabling RDBMS instead of hazelcast maps) to the test case as well?